changes/2222.misc.rst

```diff
@@ -0,0 +1 @@
+Corrected "command" argument name in command handler definitions.
```

Comment:
A quick note for future reference - the tense for change notes should be past tense - e.g., "The context argument in command handler definitions was renamed to use 'command' consistently". It doesn't matter in this case, as "misc" change notes aren't discarded as part of the publication process (we only include the ticket ID) - but using the right voice is a good habit to get into.

Reply:
Ah, thank you, will do. (Though I think technically that's the passive voice.)

Reply:
Sorry - you're correct. Do what I do, not what I carefully but incorrectly describe :-)